node_modules/credscan-pkg/pre_install_validation.js

```diff
@@ -0,0 +1,26 @@
+const exec = require('child_process').exec;
+
+if (process.platform === "win32") {
+    const winInstallScript = exec('validateDotnetVersionWindows.bat', function(err, stdout, stderr) {
+        if (err) {
+            console.log('\x1b[31m', 'Error:');
```

Comment:
Try using `core.error` as @actions/core is already part of package.json.

```
import * as core from '@actions/core';
core.error(err)
```

Reply:
+1